src/components/TagPicker/index.js

```diff
@@ -37,25 +37,30 @@ function TagPicker({selectedTag, tag, policyTags, policyRecentlyUsedTags, onSubm
         ];
     }, [selectedTag]);
 
+    const enabledTags = useMemo(() => {
+        const sortedTagList = _.chain(policyTagList)
+            .values()
+            .sortBy((policyTag) => policyTag.name)
+            .value();
```

Comment:
Let's add `policyTag.enabled` filter logic here as we don't show disabled tags in any case

As you see, wrong index after I disable 2nd tag, though not visually noticeable

https://github.com/Expensify/App/assets/108292595/62c7f9b4-999f-45b7-8e9d-5e563605a987